storage/innobase/log/log0recv.cc

```diff
@@ -3845,9 +3863,52 @@ byte *recv_dblwr_t::find_page(const page_id_t page_id,
   return result;
 }
 
+uint32_t recv_dblwr_t::find_first_page(const char* name, pfs_os_file_t file)
+{
+  uint32_t space= 0;
+  os_offset_t file_size= os_file_get_size(file);
+  if (file_size == (os_offset_t) -1) return space;
```

Comment:
I think that it would be clearer to break the line before `return`, and maybe even change it to `return 0` to make it stand out more. A `return 0` or `return space` should make no difference to the generated code, thanks to common subexpression elimination at a high-level stage in every compiler.

Alternatively, we could negate the condition and put the entire `for` loop inside `if (file_size != os_offset_t(-1))`.